pkg/sql/distsqlrun/algebraic_set_op.go

```diff
@@ -37,9 +35,20 @@ type algebraicSetOp struct {
 	opType                  AlgebraicSetOpSpec_SetOpType
 	ordering                Ordering
 	types                   []sqlbase.ColumnType
-	datumAlloc              *sqlbase.DatumAlloc
+	datumAlloc              sqlbase.DatumAlloc
+
```

Comment:
I assume this break means that everything below is runtime state - could you please add a comment that makes this explicit? We started doing this explicitly (with a separate struct - but even a comment will be helpful) in planNodes and it makes understanding the code a lot more straightforward.